### ISSUE TITLE
Clearance data - focused array compounds

### ISSUE THREAD

**Issue by danaklug** [opensourceantibiotics/Series-2-Diarylimidazoles#13]
As part of optimizing the clearance of OSA_000822, it would be great to get some clearance data on the related analogs synthesized by @AlvaroLor. A few questions so that we can get this off the ground:

- @AlvaroLor Are there samples of focused array compounds at UNC that could be sent to Monash?

- How much compound is required for clearance assays?

- Are these compounds accessible currently and is Monash able to receive them?

**Comment by mattodd** [comment]
Yes, it'd be very good to secure some of these data. The [very high intrinsic clearance](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Metabolic-Clearance-Data) of the main compound is a problem, and it'd important to know whether such data were obtained for other compounds, and whether we could easily ship to Monash for evaluation. @AlvaroLor - who's the best person to contact about supplies at Chapel Hill? Do you know if CLint was measured for other members of the series (ping @TwistedSwisster )? If there are samples ([I'd suggest actives and inactives maybe?](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Potency-Evaluation)) I can reach out to Monash, who I think are not yet on Github.

**Comment by AlvaroLor** [comment]
The entire compound library is at UNC and there are at least 15 mg of each compound. I think the clearance studies are typically performed by using a range of drug concentrations, so I would say that a couple of mg is more than enough, although is better to ask Monash what do they need. Could you please contact with them? I can contact with people from the UNC lab to help us with the shipment.
On the other hand, I think MRSA MIC values were the only measurements done with CO-ADD but @TwistedSwisster knows it better.
Hopefully, we will be able confirm/compare the predicted metabolic liabilities with the clearance in vitro values!

**Comment by mattodd** [comment]
That's great, thank you for checking whether you still have samples for shipping. I will reach out to Monash to double-check that they are OK with testing clearance. Can you (@AlvaroLor ) ot @TwistedSwisster confirm that no other compounds in this series were measured for microsomal clearance, so that we will be asking Monash for new measurements?

**Comment by danaklug** [comment]
@TwistedSwisster has kindly provided the metabolism reports, which include clearance data on **SB-*400868**, **OSA_000813**, and **OSA_000822**. These have been [uploaded](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/tree/master/Sources%20of%20Data) and a summary has been posted on the [wiki](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Metabolic-Clearance-Data)

**Comment by danaklug** [comment]
For easy reference, here is the summary of clearance data we have on focused array compounds (also on the wiki, linked above): 

![FA_metabolism_summary](https://user-images.githubusercontent.com/53487492/85420429-15cb6200-b56b-11ea-9b5e-6dc91ec07a04.PNG)

Looking at the clearance data for **SB-400868** and **OSA_000813**, both are fairly highly cleared (although more stable than **OSA_000822**), which indicates that the benzothiophene is not the only metabolic liability of this series. I ran both compounds through SMARTCyp; the results (below) indicate that the methylenes are likely to be labile (this is also supported by the MetaSite and SMARTCyp analyses of **OSA_000822**). 

![FA_SMARTCyp](https://user-images.githubusercontent.com/53487492/85420532-3bf10200-b56b-11ea-96c8-6a0e82c616fb.PNG)

A few suggestions for future testing: 

- If Monash provide HLM data, we should send **SB-400868**, **OSA_000813**, and **OSA_000822** as this would be new and valuable information.

- I think that **OSA_000814** and **OSA_000821** would also be interesting to test at Monash. Although **814** isn’t particularly active, clearance data would give a good idea of the role of the core methylenes when compared to **SB-400868**. **821** is only slightly less active than the lead thiophene **822** and may be an interesting alternative.

![image](https://user-images.githubusercontent.com/53487492/85420724-82466100-b56b-11ea-8904-a427e3a73f1d.png)

**Comment by mattodd** [comment]
This is really good @danaklug . The deal we have in the small grant we won to look at this series is that there are funds for 9 in vitro panels at Monash ("Kinetic solubility at pH 2 and 6.5, logD determination and metabolic stability in liver microsomes (human, rodent) for 9 compounds"). So if we sent these five, we'd have space for 4 more, which is OK from my perspective. Note also we have funds for one in vivo, at the end, "In vivo rat pharmacokinetics (IV/PO) including formulation and LCMS development, collection of plasma concentration vs. time profile (24 h) and determination of PK properties for 1 compound". Maybe we should acquire rat microsomal data as a result? @TwistedSwisster @AlvaroLor did you have a sense of whether mouse or rat is preferable, given the likely animal we might use to assess in vivo PK? ping @rhanson1046

**Comment by drc007** [comment]
@danaklug @mattodd I would expect the benzofuran of OSA_000821 to be another metabolic hotspot. Do we have any with an electron-deficient ring?

**Comment by danaklug** [comment]
@drc007 No, not really. With 821 I was hoping we would get an idea of the clearance of another active compound. However, I don't think that data would be crucial to decisions going forward, so we could always table that one and save the slot.

**Comment by drc007** [comment]
@danaklug benzothiophene‐1,1‐diones are known, and a quick search suggests a number of potential building blocks are available. Not a structural class I am that familiar with.
![sulphone](https://user-images.githubusercontent.com/7080516/86494468-26e75080-bd6d-11ea-905f-684d6f7d2b4d.png)

**Comment by AlvaroLor** [comment]
@mattodd We have used rat, mouse, rabbit or human liver microsomes to evaluate the metabolic stability of different small molecule libraries, but we selected them based on prior studies. However, there is not any previous stability study available for this compound family. On the "Surrounding Chemical Space and Literature" post, @danaklug listed some papers related with 4,5-diaryl-imidazoles, most of them are pure synthesis but in this recent one: [https://pubs.acs.org/doi/abs/10.1021/acsmedchemlett.9b00552?journalCode=amclct&quickLinkVolume=11&quickLinkPage=172&selectedTab=citation&volume=11](url) they evaluated PK properties and metabolic liabilities of diaryl imidazoles using mouse, rat, human and dog microsomes. So I think either rat or mouse are fine, but if the in vivo studies are going to be performed with rats, it makes sense to use rat microsomes as the preferred choice.

**Comment by edwintse** [comment]
UPDATE: The following 5 compounds have now all been sent to Monash for clearance measurements. Results will be posted when received.

![Untitled Wiley Document-29](https://user-images.githubusercontent.com/18062981/91881425-5f29e300-ec79-11ea-82d1-ebf77225f0c8.jpeg)

**Comment by rhanson1046** [comment]
Thanks for the update and good luck.
Bob Hanson
________________________________

**Comment by mattodd** [comment]
Karen White has asked "mouse or rat"? It makes sense I think to go for rat, given that that is the _in vivo_ experiment we're aiming for. @AlvaroLor agrees. Any advocates for mouse data?

**Comment by drc007** [comment]
@mattodd Priorities should be in vivo efficacy species and probable species to be used in safety studies, so rat seems appropriate.

**Comment by danaklug** [comment]
Monash results have been posted to the [wiki](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Metabolic-Clearance-Data) and will be discussed in today's [project meeting](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/29) (25 Sept. 2020).

**Comment by rhanson1046** [comment]
Thanks for the update. I have downloaded the MRSA data and the drug metabolism results and will give some thought to them.
R. Hanson
Department of Chemistry and Chemical Biology
Northeastern University
________________________________

**Comment by mattodd** [comment]
In thinking about which compound to send for evaluation of metabolite formation (#29) our friends at Hypha asked whether we have done any metasite predictions on the compound we'd like to ship (OSA821 - the benzofuran). I'm not sure we have, have we? @drc007 @danaklug @AlvaroLor @TwistedSwisster ? They mentioned a paper we might want to check out related to this (https://www.mdpi.com/2073-4344/10/1/62). The proof will be in the pudding (is the furan being oxidised, or the pyridyl?), but chance favours the prepared mind, I guess.
